packages/dds/sequence/api-report/sequence.api.md

```diff
@@ -592,14 +592,7 @@ export abstract class SharedSegmentSequence<T extends ISegment> extends SharedOb
     // (undocumented)
     getRangeExtentsOfPosition(pos: number): {
         posStart: number | undefined;
-        posAfterEnd: number | undefined; /**
```

Comment:
Interesting. This change seems like an improvement, but I'm curious why it happened here.

Reply:
When we updated API extractor it started picking up some end of line comments in the reports sometimes. I think this was one of those. Not sure why that changed in this PR though. Maybe it has a bug that source location depended (ex: end of line at end of buffer kind of thing) and this moved the code around a little fixing it?

Reply:
Same diff showed up in https://github.com/microsoft/FluidFramework/pull/21067 when I merge main into it and rebuilt, so it seems like an odd state on main, not an issue with this change.